src/d3a/models/strategy/storage.py

```diff
@@ -167,6 +171,16 @@ def _validate_break_even_points(break_even):
         if any(break_even_point.buy < 0 or break_even_point.sell < 0
                for _, break_even_point in break_even.items()):
             raise ValueError("Break even point should be positive energy rate values.")
+        if any(break_even_point.sell > initial_selling_rate
+               for break_even_point in break_even.values()):
+            raise ValueError(f"Break even point (selling) should be lower equal "
```

Comment:
Trivial, but you could add at least the initial selling rate value in the exception text. That way the user will be able to see its value. Same for the initial buying rate exception below.

Reply:
I was thinking about that but got distracted. Will add it. Thanks!